src/core/public/chrome/global_search/global_search_service.ts

```diff
@@ -46,16 +47,127 @@ export interface GlobalSearchCommand {
    * @param value search query
    * @param callback callback function when search is done
    */
-  run(value: string, callback?: () => void): Promise<ReactNode[]>;
+  run(value: string, callback?: () => void, abortSignal?: AbortSignal): Promise<ReactNode[]>;
 }
 
+/**
+ * @experimental
+ */
+export interface GlobalSearchSubmitCommand {
```

Comment:
Agreed to use an unified Command interface, instead of introducing a new `GlobalSearchSubmitCommand`, it should extend the existing interface `GlobalSearchCommand`